plugins/generator-1.20.1/forge-1.20.1/templates/elementinits/entityrenderers.java.ftl

```diff
@@ -44,7 +44,7 @@ package ${package}.init;
 				<#if entity.isCustomModel()>
 				event.registerEntityRenderer(${JavaModName}Entities.${entity.getModElement().getRegistryNameUpper()}.get(), ${entity.getModElement().getName()}Renderer::new);
 				<#else>
-				event.registerEntityRenderer(${JavaModName}Entities.${entity.getModElement().getRegistryNameUpper()}.get(), ThrownItemRenderer::new);
+				event.registerEntityRenderer(${JavaModName}Entities.${entity.getModElement().getRegistryNameUpper()}.get(), <#if entity.visualScale != 1>render -> new ThrownItemRenderer<>(render, ${entity.visualScale}f, true)<#else>ThrownItemRenderer::new</#if>);

```

Comment:
Scale parameter currently only applies to vanilla renderer but not to custom models.

Comparing to living entity ME where scale scales all models.

It will be confusing to the user that scale parameter just won't do anything for custom models.

Make this parameter work for custom models too.

Applies to both versions